src/m365/flow/commands/flow-export.ts

```diff
@@ -87,9 +87,9 @@ class FlowExportCommand extends AzmgmtCommand {
         }
 
         const requestOptions: any = {
-          url: `${this.resource}providers/${formatArgument === 'json' ?
-            `Microsoft.ProcessSimple/environments/${encodeURIComponent(args.options.environment)}/flows/${encodeURIComponent(args.options.id)}?api-version=2016-11-01`
-            : `Microsoft.BusinessAppPlatform/environments/${encodeURIComponent(args.options.environment)}/exportPackage?api-version=2016-11-01`}`,
+          url: formatArgument === 'json' ?
```

Comment:
Following the discussion @Jwaegebaert, I'm wondering whether we cannot just remove all references to `management.azure.com` in this command. We could do so by changing the BaseCommand from `AzmgmtCommand` to `PowerPlatformCommand`. That way `this.resource` would always point to the api.bap url.
Could you try that out to see if that keeps the functionality working as expected for all exported formats?

@waldekmastykarz: please correct me if I overlook something here.

Reply:
I'd say the change you propose @martinlingstuyl is outside the scope of this issue and we should take it separately because we need to validate that it doesn't introduce any breaking behavior. Let's keep this change as-is and take what you suggested in a separate issue.

Reply:
This is indeed only for the zip-export

Reply:
So, would you by any change like to create an issue for the updating of the base class @Jwaegebaert?